src/test/java/eu/elixir/ega/ebi/it/res/SSLUtilities.java

```diff
@@ -0,0 +1,377 @@
+package eu.elixir.ega.ebi.it.res;
```

Comment:
Maybe a little explanation about why this is needed and what it does in the README and a litle description of the structure. Also the test does http no https request so I am not sure why this is needed

Reply:
You are right here probably we don't need this class so I have removed it.